main.py

```diff
@@ -34,6 +35,11 @@ def create_menu():
 def add_book():
     new_book = ui.get_book_info()
     new_book.save()
+
+def delete_book():
+    search_term = ui.ask_question('Enter book ID:   '   ) 
+    matches = store.book_search(search_term)
```

Comment:
Where does the book get deleted? there's a delete method in the bookstore to call